seacrowd/sea_datasets/typhoon_yolanda_tweets/typhoon_yolanda_tweets.py

```diff
@@ -0,0 +1,139 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION, Tasks.DEPENDENCY_PARSING]
+
+_CITATION = """\
+@misc{imperial2019sentiment,
+      title={Sentiment Analysis of Typhoon Related Tweets using Standard and Bidirectional Recurrent Neural Networks}, 
+      author={Joseph Marvin Imperial and Jeyrome Orosco and Shiela Mae Mazo and Lany Maceda},
+      year={2019},
+      eprint={1908.01765},
+      archivePrefix={arXiv},
+      primaryClass={cs.NE}
+}
+"""
+
+_DATASETNAME = "typhoon_yolanda_tweets"
+
+_DESCRIPTION = """\
+The dataset contains annotated typhoon and disaster-related tweets in Filipino collected before, during, 
+and after one month of Typhoon Yolanda in 2013. The dataset has been annotated by an expert into three 
+sentiment categories: positive, negative, and neutral.
+"""
+
+_HOMEPAGE = "https://github.com/imperialite/Philippine-Languages-Online-Corpora/tree/master/Tweets/Annotated%20Yolanda"
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_ROOT_URL = "https://raw.githubusercontent.com/imperialite/Philippine-Languages-Online-Corpora/master/Tweets/Annotated%20Yolanda/"
+_URLS = {"train": {-1: _ROOT_URL + "train/-1.txt", 0: _ROOT_URL + "train/0.txt", 1: _ROOT_URL + "train/1.txt"}, "test": {-1: _ROOT_URL + "test/-1.txt", 0: _ROOT_URL + "test/0.txt", 1: _ROOT_URL + "test/1.txt"}}
+
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+# TODO: Name the dataset class to match the script name using CamelCase instead of snake_case
+class TyphoonYolandaTweets(datasets.GeneratorBasedBuilder):
```

Comment:
remove TODO since it has been done
```suggestion
class TyphoonYolandaTweets(datasets.GeneratorBasedBuilder):
```

Reply:
removed in the new commit. thanks